gensim/models/fasttext.py

```diff
@@ -37,20 +37,20 @@
     ['human', 'interface', 'computer']
     >>> print(len(common_texts))
     9
-    >>> model = FastText(size=4, window=3, min_count=1)  # instantiate
+    >>> model = FastText(vector_size=4, window=3, min_count=1)  # instantiate
```

Comment:
Not sure what you mean. #1777 changed many, but not all, uses of `size` to represent a 2Vec model's vector-size to `vector_size`. Consistency required either rolling back to consistent use of `size`, or using `vector_size` everywhere. I went with the latter for clarity.

Reply:
Yes, that's what I meant. The motivation for this (top-level) API change. Consistency is a good motivation, I wasn't aware.

Which models were affected, besides fastText? Can we deprecate `size` = keep as alias for at least a few releases?
I hate breaking backward compatibility, especially where it seems wanton, with no benefit besides aesthetics.

Reply:
Keeping `size` as an alias will require lots of method signature-expansion, and start-of-method conditionals to patch old parameter into new places - low value typing for us to 1st add that code, then remove it after "a few releases". 

But a major-release that's already forcing many user code changes (as elsewhere, where lots of things that have been generating deprecation warnings "will be removed in 4.0.0" will now be gone breaking old user code) is the ideal place to shift this cost to users, with a single bullet point in the release notes, "Word2Vec, FastText, Doc2Vec and related classes now consistently use `vector_size` instead of `size`."